### DISCUSSION TITLE
Model and sub Model in one storage controller

### DISCUSSION THREAD

**Discussion by mquevedob** [savannabits/jig#16] [Q&A]
hello 

I would like to know if the following is possible in JIG

Lets say I have a model named Order and then another model named OrderItem

Order will have id and name, and OrderItem will have id, name and order_id

how should I configure the migration for orders and order_items tables so that JIG will create a CRUD capable to allow me to create an order and add the order items in the same form?

Regards,
Martin

**Comment by coolsam726** [comment]
Hi @mquevedob ,

At the moment it is not possible to generate such code, but I have used the pattern severally in many projects. You only need to do a few modifications to make it work. I will share the code a bit later to guide you on this.

**Reply by mquevedob** [reply]
thank you !
